libs/librepcb/project/boards/items/bi_netsegment.cpp

```diff
@@ -194,6 +194,24 @@ int BI_NetSegment::getViasAtScenePos(const Point&    pos,
   return count;
 }
 
+int BI_NetSegment::getNetPointNextToScenePos(const Point&         pos,
+                                             const GraphicsLayer* layer,
+                                             BI_NetPoint** bestMatch,
+                                             UnsignedLength* maxDistance) const
+    noexcept {
+  foreach (BI_NetPoint* netpoint, mNetPoints) {
+    if (netpoint->isSelectable() && ((!layer) || (netpoint->getLayerOfLines() == layer))) {
+      UnsignedLength distance = netpoint->distance(pos);
+      if (! maxDistance || distance < *maxDistance) {
+        *maxDistance = distance;
+        *bestMatch = netpoint;
+      }
+    }
+  }
+  return 0;
```

Comment:
This looks like the return value is actually not needed :wink: Maybe better return `bestMatch` instead of using an out-parameter for it? Or least change the return type to `void`.